### PR TITLE
Include logged-in user in the custom-email logs

### PR DESCRIPTION
**Pull request by vasantvohra**

### CODE REVIEW

File: indico/modules/events/registration/controllers/management/reglists.py [hunk 1]
```diff
@@ -229,7 +229,8 @@ def _send_emails(self, form):
                                    attachments=attachments)
             signals.core.before_notification_send.send('registration-custom-email', email=email,
                                                        registration=registration, form=form)
-            send_email(email, self.event, 'Registration', log_metadata={'registration_id': registration.id})
+            send_email(email, self.event, 'Registration', user=session.user,
+                       log_metadata={'registration_id': registration.id})
```
Comment (ThiefMaster) [inline indico/modules/events/registration/controllers/management/reglists.py:233]: You can use `registration.log(...)` which automatically passes the `log_metadata`

Comment (vasantvohra) [reply indico/modules/events/registration/controllers/management/reglists.py:233]: but this is `send_email` which uses `_log_email` that uses `event.log(...)`

Comment (ThiefMaster) [reply indico/modules/events/registration/controllers/management/reglists.py:233]: oh right, nevermind

File: indico/modules/events/registration/controllers/management/reglists.py [hunk 2]
```diff
@@ -229,7 +229,8 @@ def _send_emails(self, form):
                                    attachments=attachments)
             signals.core.before_notification_send.send('registration-custom-email', email=email,
                                                        registration=registration, form=form)
-            send_email(email, self.event, 'Registration', log_metadata={'registration_id': registration.id})
+            send_email(email, self.event, 'Registration', user=session.user,
```
Comment (ThiefMaster) [inline indico/modules/events/registration/controllers/management/reglists.py:232]:
```suggestion
            send_email(email, self.event, 'Registration', session.user,
```

we pass it as a posarg everywhere else

Comment (vasantvohra) [reply indico/modules/events/registration/controllers/management/reglists.py]: done

**Pull request merged**